### ISSUE TITLE
warning when compiling with clang++

### ISSUE THREAD

**Issue by predat** [CGRU/cgru#206]
I've 2 warning when compiling afanasy that i can't manage myself:
first:
```
cgru/afanasy/src/server/monitoraf.cpp:144:56: warning: if statement has empty body [-Wempty-body]
                                        if( setListening( job_id, block, task, subscribe));
```
it seem that L145 is not part of the 'if'...

second:
```
cgru/afanasy/src/server/sysjob.cpp:474:7: warning: equality comparison result unused [-Wunused-comparison]
        m_id == AFJOB::SYSJOB_ID;
        ~~~~~^~~~~~~~~~~~~~~~~~~
/home/predat/Documents/dev/cgru/afanasy/src/server/sysjob.cpp:474:7: note: use '=' to turn this equality comparison into an assignment
        m_id == AFJOB::SYSJOB_ID;
             ^~
             =
1 warning generated.
```
???

**Comment by timurhai** [comment]
wow!
These are bugs!
Bit not critical.
 - There should not be ` ; ` after ` if ` condition ( now it can subscribe 2 times on the same task - but it should not )
 -  There should be ` = ` instead of ` == `. System job ID should be `AFJOB::SYSJOB_ID` ( = 1 ).
You can fix them too.

**Comment by predat** [comment]
Fixed 7583ef8
